RPIPs/RPIP-49.md

```diff
@@ -118,7 +121,11 @@ The tokenomics rework package will likely be split between two protocol upgrades
   * 1.5ETH minimum bond. 
 * [RPIP-44: Forced exits](RPIP-44.md)
 * [RPIP-46: Universal Adjustable Revenue Split](RPIP-46.md)
-  * Heuristic adjustments.
+  * Split may add a new share for all RPL, depending on outcome of Revenue Share vote
+  * May add in a semi-automated system of controlling voter_share, depending on outcome of Revenue Share vote
+  * RPL inflation reduced to 1.5%  (from 5%)
+  * No more RPL issuance rewards
+* RPL Value Capture based on vote outcome - [RPL Burn](RPIP-45.md) / [RPL Buy & LP](RPIP-50.md) / Increased share to voting Node Operators.
```

Comment:
I like this updated style/section... Last comment on same topic ("voting Node Operators" ok?)

Reply:
Increased share to vote-eligible RPL staked in megapools